### ISSUE TITLE
[BUG] Minor bug in Introduction_to_TF24_IVIM-MRI_CodeCollection_github_and_IVIM_Analysis_using_Python.ipynb

### ISSUE THREAD

**Issue by oliverchampion** [OSIPI/TF2.4_IVIM-MRI_CodeCollection#83]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the bug

in In 26, D and D* are swapped when plotted. I would normally change it, but do not have the capability of editing ipynb right now.

### Screenshots [optional]

![downloaden](https://github.com/user-attachments/assets/6e6c9964-dd9f-4044-8c48-98867c238804)

### Steps To Reproduce

Visit: https://github.com/OSIPI/TF2.4_IVIM-MRI_CodeCollection/blob/main/doc/Introduction_to_TF24_IVIM-MRI_CodeCollection_github_and_IVIM_Analysis_using_Python.ipynb

### Expected behavior

D and D* should be swapped

### Are you working on this?

No

**Comment by Mag-D-Anas** [comment]
@etpeterson, Can I do it while combining it with importing from Zenodo, as I mentioned before?

**Comment by Mag-D-Anas** [comment]
![Image](https://github.com/user-attachments/assets/de5eed47-dbe3-4723-9393-c6fc0582db2a)
does this mean this issue should be closed? seems correct on my machine

**Comment by etpeterson** [comment]
Yeah, that looks correct now. Thanks for checking, I'll close this.
